.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+version: 2
+
+# This is not a CircleCI key. I'm just defining shared YAML keys here,
```

Comment:
Seems like a really small comment, but it'd be good to move this comment next to `job_defaults`. I scratched my head over the yaml syntax below, and didn't realize this comment was meant to explain it.